### PR TITLE
Creates CONTRIBUTING.md, closes #5

### LINKED ISSUE #5: Add instructions to help develop
In #4 you linked to a new branch with the `initialSortKey` feature and invited me to try it out. Thanks! That was great.

I ran into a little trouble with getting this new code into my existing project (where I had previously just done `npm install react-table`). 

Here's what I did to make it work for now:

1. fork this repo on GitHub
2. clone my fork locally to `/Users/dsernst/Documents/react-table/`
3. from my fork: `git checkout origin/#4-configurable-sort-column`
4. from this detached HEAD, checkout into a new local branch: `git checkout -b \#4-configurable-sort-column`
5. `npm pack` to make it into a local tarball
6. Went back to my project that uses `react-table` and then: `npm install /Users/dsernst/Documents/react-table/react-table-1.3.0.tgz`

I don't know if this is how you're managing local development, but whatever it is, it might be helpful to add some instructions to the README to lower the barrier to entry.

In any case, thanks again for being so responsive with #4.

### PR DESCRIPTION
**Pull request by dsernst**
Adds a `CONTRIBUTING.md` file with 3 different methods to test local changes to `react-table`

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+Here's one approach to test out local changes, e.g. for a new branch called `#4-configurable-sort-column`
+
+1. fork this repo on GitHub
+2. clone your fork to your local machine
+3. from your fork: `git checkout origin/#4-configurable-sort-column`
+4. from this detached HEAD, checkout into a new local branch: `git checkout -b \#4-configurable-sort-column`
+5. `npm pack` to make it into a local tarball
+6. From your project that uses `react-table`: `npm install /your/local/path/react-table/react-table-1.3.0.tgz`
+
+------------------------
```
Comment (NickTomlin) [inline CONTRIBUTING.md]: Looking at this, I wonder if we should just advocate using `npm link` since it will simplify things.

**Comment by NickTomlin** [comment]
@dsernst this is great, thanks for taking the time to work on this! I think we should switch to just using npm link in the instructions for clarity. What do you think? Once we've made a decision on that i'll merge it in.

**Comment by dsernst** [comment]
Makes sense! :+1:

On Tuesday, June 9, 2015, Nick Tomlin <notifications@github.com> wrote:

> @dsernst <https://github.com/dsernst> this is great, thanks for taking
> the time to work on this! I think we should switch to just using npm link
> in the instructions for clarity. What do you think? Once we've made a
> decision on that i'll merge it in.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/NickTomlin/react-table/pull/6#issuecomment-110373510>.
>


-- 
--
David Ernst

**Comment by NickTomlin** [comment]
Great, i'll merge this in and make the edit. Thanks much!

**Pull request merged**